modules/tinymce/src/themes/silver/test/ts/browser/editor/contextmenu/MobileContextMenuTest.ts

```diff
@@ -163,31 +159,6 @@ describe('browser.tinymce.themes.silver.editor.contextmenu.MobileContextMenuTest
     await pWaitForAndCloseDialog(editor);
   });
 
-  it('TBA: Test context menus on image inside a table', async () => {
```

Comment:
This shouldn't have been removed as there was still some parts here that aren't imagetools related.

- [ ] Revert this and only remove the imagetools specific logic (e.g `selectors.editimage`)